src/importer/Importer.js

```diff
@@ -0,0 +1,21 @@
+import { readFile } from 'fs';
```

Comment:
If I remember it right, you must implement sync and async import methods

Reply:
and these methods shouldn't be static but own class methods reachcable only when creating class instance